src/connectors/hits-per-page/connectHitsPerPage.ts

```diff
@@ -190,6 +192,7 @@ You may want to add another entry to the \`items\` option with this value.`
 
         renderFn(
           {
+            ...this.getWidgetRenderState(initOptions),
             items: transformItems(normalizeItems(state)),
             refine: connectorState.setHitsPerPage,
             createURL: connectorState.createURLFactory(helper.state),
```

Comment:
these functions aren't needed anymore, since they are now part of getWidgetRenderState

Reply:
Ok, I was wondering after I splitted the functions, thanks